mev-relay-rs/src/relay.rs

```diff
@@ -448,6 +451,28 @@ impl Relay {
         state.auctions.insert(auction_request, auction_context);
         Ok(())
     }
+
+    fn verify_blinded_block_signature(
+        &self,
+        signed_block: &mut SignedBlindedBeaconBlock,
+    ) -> Result<(), ethereum_consensus::Error> {
+        let proposer = self
+            .validator_registry
+            .get_public_key(signed_block.message().proposer_index())
+            .unwrap();
+        let slot = signed_block.message().slot(); // signed_block.message().slot() + 1 ???;
+        let genesis_validators_root = [0u8; 4].hash_tree_root().map_err(ConsensusError::from)?; // Should `compute_consenus_signing_root()` take Option<&Root> ?
+        let mut block =
+            signed_block.message_mut().hash_tree_root().map_err(ConsensusError::from)?;
```

Comment:
if you look at `compute_consensus_signing_root` you'll see it is already taking the `hash_tree_root` of the data you pass in, so the idea here is that you just pass in the SSZ data you want to get the signing root of

in this case, you should just be able to do this:
```suggestion
        let mut block =
            signed_block.message_mut();
```